src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/dashboard/MonitoringViewConstants.java

```diff
@@ -1,34 +1,37 @@
 package edu.kit.pse.osip.monitoring.view.dashboard;
 
+import edu.kit.pse.osip.core.SimulationConstants;
 import javafx.scene.paint.Color;
 import javafx.stage.Screen;
 
 /**
  * Constants for the monitoring view, appearing in more than one class.
  * 
  * @author Martin Armbruster
- * @version 1.4
+ * @version 1.5
  */
 public final class MonitoringViewConstants {
     /**
      * Threshold (in %) when a overflow alarm triggers.
      */
-    public static final Float OVERFLOW_ALARM_THRESHOLD = Float.valueOf(95);
+    public static final Float OVERFLOW_ALARM_THRESHOLD = Float.valueOf(0.95f);
     
     /**
      * Threshold (in %) when an underflow alarm triggers.
      */
-    public static final Float UNDERFLOW_ALARM_THRESHOLD = Float.valueOf(5);
+    public static final Float UNDERFLOW_ALARM_THRESHOLD = Float.valueOf(0.05f);
         
     /**
      * Threshold (in °K) when an alarm for overheating triggers.
      */
-    public static final Float TEMPERATURE_OVERHEATING_THRESHOLD = Float.valueOf(678.5f);
+    public static final Float TEMPERATURE_OVERHEATING_THRESHOLD = Float.valueOf(SimulationConstants.MIN_TEMPERATURE
+            + 0.95f * (SimulationConstants.MAX_TEMPERATURE - SimulationConstants.MIN_TEMPERATURE));
     
     /**
      * Threshold (in °K) when an alarm for undercooling triggers.
      */
-    public static final Float TEMPERATURE_UNDERCOOLING_THRESHOLD = Float.valueOf(291.5f);
+    public static final Float TEMPERATURE_UNDERCOOLING_THRESHOLD = Float.valueOf(SimulationConstants.MIN_TEMPERATURE
```

Comment:
Also ich habe in der Simulation feste Werte genommen, die m.M.n. mehr Sinn ergeben: Als Untergrenze 0°C (damit kein Wasser gefriert) und als Obergrenze 100°C (damit nichts kocht). Diese Werte gelten natürlich nur für Wasser, aber bei den Flüssigkeiten denkt man vielleicht auch erst mal an Wasser.